src/entry.js

```diff
@@ -56,19 +76,32 @@ module.exports = class Entry {
       })
   }
 
-  // Returns a boolean
-  // Example:
-  //   const hasChild = Entry.hasChild(entry1, entry2)
-  //   true|false
+  /**
+   * Check if an entry has another entry as its child
+   * @param {Entry} [entry1] Entry to check from
+   * @param {Entry} [entry2] Child
+   * @returns {boolean}
+   */
   static hasChild(entry1, entry2) {
     return entry1.next.includes(entry2.hash)
   }
 
-  // Returns a boolean
-  // Example:
-  //   const equal = Entry.compare(entry1, entry2)
-  //   true|false
+  /**
+   * Check if an entry equals another entry
+   * @param {Entry} a
+   * @param {Entry} b
+   * @returns {boolean}
+   */
   static compare(a, b) {
     return a.hash === b.hash
   }
+
+  /**
+   * @alias compare
+   */
+  static isEqual(a, b) {
+    return a.hash === b.hash
```

Comment:
Gonna rework this and drop `compare` altogether and just keep the equality check.